packages/builders/scripts/testing/replace-feeDistributor-short.js

```diff
@@ -3,7 +3,7 @@ import { getManifestForReplaceFeeDistributor } from '@agoric/inter-protocol/src/
 
 /** @type {import('@agoric/deploy-script-support/src/externalTypes.js').CoreEvalBuilder} */
 export const defaultProposalBuilder = async (_, opts) => {
-  console.log('OPTS', opts);
+  console.log('feeDist  OPTS', opts);
```

Comment:
what's this `-short.js` file for? a `@file` comment would be nifty

Reply:
added a comment.